src/System.Console/tests/ConsoleEncoding.cs

```diff
@@ -11,6 +11,8 @@
 
 public partial class ConsoleEncoding : RemoteExecutorTestBase
 {
+    public static bool ConsoleEncodingAvailable => !PlatformDetection.IsUap || PlatformDetection.IsWindows10RS3OrGreater;
```

Comment:
@danmosemsft - switch is working but console encoding is still not working on RS3 (test are failing with invalid handle)